projects/ng-components-docs/src/app/examples/components/pagination/pagination.module.ts

```diff
@@ -0,0 +1,57 @@
+import { NgModule } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { ExamplePageComponent } from "../../../components/example-page/example-page.component";
+import { ExampleListComponent } from "../../../components/example-page/example-list/example-list.component";
+import { ApiListComponent } from "../../../components/example-page/api-list/api-list.component";
+import { Route } from "@angular/router";
+import { PaginationComponent } from "./examples/pagination/pagination.component";
+import { PaginationModule as Pagination } from "../../../../../../../lib/src/pagination";
+
+export const ROUTES: Array<Route> = [
+    { path: "", pathMatch: "full", redirectTo: "examples" },
+    {
+        path: "",
+        component: ExamplePageComponent,
+        children: [
+            {
+                path: "examples",
+                component: ExampleListComponent,
+                children: [
+                    {
+                        path: "pagination",
+                        component: PaginationComponent,
+                        data: {
+                            title: "Pagination",
+                            description: "Additional description for example (optional)",
+                            sources: [
+                                {
+                                    name: "pagintaion.component.html",
+                                    src: require("!raw-loader!./examples/pagination/pagination.component.html"),
+                                    lang: "markup",
+                                },
+                                {
+                                    name: "pagination.component.ts",
+                                    src: require("!raw-loader!./examples/pagination/pagination.component.ts"),
```

Comment:
I don't think you need to mention `raw-loader`

Reply:
I am not sure if it will be able to parse the file .ts to generate the api as intended. Neither am i aware of the consequences of removing the ra-loader. I don't know how this configuration handles the api generation, but going by the other examples, i will not be in a heist to parse it without the raw-loader.